engine/src/main/java/org/camunda/bpm/engine/impl/bpmn/parser/BpmnParse.java

```diff
@@ -2837,6 +2838,8 @@ public TaskDefinition parseTaskDefinition(Element taskElement, String taskDefini
     // Activiti custom extension
     parseUserTaskCustomExtensions(taskElement, activity, taskDefinition);
 
+    taskDefinition.addBuiltInTaskListener(TaskListener.EVENTNAME_UPDATE, SetLastUpdatedTaskListener.INSTANCE);
```

Comment:
Setting this as a task listener is a rather fragile construct in my view. Task listeners can be influenced by users with custom parse listeners for example. Although your listener will currently be the first one to be executed (before custom listeners), it might be more stable to add this rather to the `triggerUpdateEvent` method in the `TaskEntity`. You have to be careful with the current state of the entity then, unfortunately. The code you are currently using in the listener should also work in that method. If this poses a problem in any case, we should find out why and adjust the code accordingly if necessary. Having it in that method appears more stable to me.